test/re/frida/MethodTest.java

```diff
@@ -54,6 +58,131 @@ public void replacementPropagatesExceptions() {
         thrown.expectMessage("Already dead");
         badger.die();
     }
+    
+    @Test
+    public void interfaceCannotBeInstantiated() {
+        loadScript("var X509TrustManager = Java.use('javax.net.ssl.X509TrustManager');" +
+                "try {" +
+                "  var tm = X509TrustManager.$new();" +
+                "} catch (e) {" + 
+                "  send('couldnt create trustmanager');" + 
+                "}" + 
+                "send('ok');"
+                );
+        assertEquals("ok", script.getNextMessage());
+    }
+    
+    @Test
+    public void TestClassForNameOrig() {
```

Comment:
Also, this test should be named like the other tests, without a `test` prefix, and communicating the expected outcome.